webui/react/src/pages/ExperimentDetails/ExperimentTrials.table.tsx

```diff
@@ -54,4 +54,10 @@ export const columns: ColumnType<TrialItem>[] = [
     key: 'checkpoint',
     title: 'Checkpoint',
   },
+  {
+    align: 'right',
+    className: 'fullCell',
+    key: 'actions',
+    title: '',
+  },
```

Comment:
**suggestion:** add `fixed: 'right'` and `width: 40` to persist the overflow action regardless of the viewport width?